bHY6yh-de2539/cod-vyg67d_bHY6yh-de2539.xml

```diff
@@ -0,0 +1,280 @@
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 10</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 10</title>
+               <date when="2023-06-12">June 12, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="V" n="cod-pu6y7d"> Vatican, Biblioteca Apostolica Vaticana, Vat. lat. 8175</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-12" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#O" n="12-v"/>
+            <cb ed="#O" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de2539"><!-- p1l1-r10 -->
+            <head xml:id="bHY6yh-de2539-Hd1e108">Rubrica 10</head>
+            <head xml:id="bHY6yh-de2539-Hd1e111" type="question-title">De charitate hominis ad deum</head>
+            <p xml:id="bHY6yh-de2539-d1e114">
+               De caritate 
+               <lb ed="#O"/>hominis ad deum. Rubrica. 10
+            </p>
+            <p xml:id="bHY6yh-de2539-d1e119">
+               <lb ed="#O"/><ref xml:id="bHY6yh-de2539-Rd1e116">In exodo. capitulo 20.</ref> ¶ <quote xml:id="bHY6yh-de2539-Qd1e123">Ego sum dominus ["deus" omitted] 
```

Comment:
you don't need to (and shouldn't) write [omitted] for any word that is missing. We only write this if the entire entry is missing.


```suggestion
               <lb ed="#O"/><ref xml:id="bHY6yh-de2539-Rd1e116">In exodo. capitulo 20.</ref> ¶ <quote xml:id="bHY6yh-de2539-Qd1e123">Ego sum dominus 
```